api/prontolista/testcases/tests/test_models.py

```diff
@@ -0,0 +1,26 @@
+from django.test import TestCase
+
+from model_mommy import mommy
+
+from projects.models import Project
+from ..models import TestCase as TestCaseModel
```

Comment:
Let's switch this with `from projects.models import Project`?

Reply:
Will fix this after we complete setup isort kub